com.tle.reporting.common/.classpath

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry excluding="**/.svn/*|**/_svn/*" kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
```

Comment:
Why is it all 1.7 and not 1.8 - which is the version we now target with oEQ and the Admin Console?

(This being the first occurrence, I've only mentioned it here - but same query applies to all the below.)